hpedockerplugin/file_manager.py

```diff
@@ -0,0 +1,479 @@
+import base64
+import copy
+import json
+import socket
+import string
+import sh
+import six
+from Crypto.Cipher import AES
+from threading import Thread
+
+from oslo_log import log as logging
+from oslo_utils import netutils
+
+from hpedockerplugin.cmd import cmd_createshare
+from hpedockerplugin.cmd import cmd_generate_fpg_vfs_names
+from hpedockerplugin.cmd import cmd_setquota
+from hpedockerplugin.cmd import cmd_deleteshare
+
+import hpedockerplugin.exception as exception
+import hpedockerplugin.fileutil as fileutil
+import hpedockerplugin.hpe.array_connection_params as acp
+from hpedockerplugin.i18n import _
+from hpedockerplugin.hpe import hpe_3par_mediator
+from hpedockerplugin import synchronization
+
+LOG = logging.getLogger(__name__)
+
+
+class FileManager(object):
+    def __init__(self, host_config, hpepluginconfig, etcd_util,
+                 fp_etcd_client, node_id, backend_name='DEFAULT'):
```

Comment:
@imran-ansari Why is backend_name set to 'DEFAULT', is there any use case for this?